app/views/users/show.html.erb

```diff
@@ -1 +1,17 @@
-<h1>"Here is the user`s bio"<h1>
\ No newline at end of file
+<div class="user_page">
+    <div class="user_page_username">
+        <img class="user_page_image" src="<%= @user.photo %>" alt="<%= @user.name %>">
+        <div class="user_page_detail">
+            <p> <%= @user.name %> </p>
+            <p class="user_posts_count">Number of posts: <%= @user.PostsCounter %></p>
+        </div>
+    </div>
+    <div class="user_page_bio">
+        <span class="bio_header">Bio</span>
+        <p class="user_bio"><%= @user.bio %> </p>
+    </div>
+    <div class="user_page_posts">
+        <%= render partial: "posts" %>
+    </div>
+    <button class="see_all_posts_btn">See all posts</button>
```

Comment:
`See all Posts` link should open the user's post index page. It's just a button now with no link. Kindly add a link to the `user's post index` page here.

Reply:
Thanks for the review. I added the link to the user`s post index now.